pkg/migration/runner.go

```diff
@@ -397,7 +397,7 @@ func (r *Runner) attemptMySQLDDL(ctx context.Context) error {
 	alterStmt := fmt.Sprintf("ALTER TABLE %s %s", r.migration.Table, r.migration.Alter)
 	err = utils.AlgorithmInplaceConsideredSafe(alterStmt)
 	if err != nil {
-		r.logger.Warnf("inplace DDL is not safe: %v", err)
+		r.logger.Infof("unable to use INPLACE: %v", err)
```

Comment:
I know we originally decided Warnf, but I spoke to an internal customer who was confused by this. So I am changing it slightly.